### PR TITLE
Add Fleshgod Apocalypse FA news post

### PR DESCRIPTION
**Pull request by pishifat**
links!!!!

### CODE REVIEW

File: news/2021-06-02-new-featured-artist-fleshgod-apocalypse.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### The Fool
+
+Play through any of these maps:
+
+- [Loved osu!catch difficulty spread](https://osu.ppy.sh/beatmapsets/445798) hosted by [Razor Sharp](https://osu.ppy.sh/users/3414261)
+- [Ranked osu!mania difficulty spread](https://osu.ppy.sh/beatmapsets/1301878) hosted by [Mipha-](https://osu.ppy.sh/beatmapsets/1301878)
```
Comment (Nivalyx) [inline news/2021-06-02-new-featured-artist-fleshgod-apocalypse.md:50]: `hosted by [Mipha-](https://osu.ppy.sh/beatmapsets/1301878)` --> `hosted by [Mipha-](https://osu.ppy.sh/users/5767941)`

File: news/2021-06-02-new-featured-artist-fleshgod-apocalypse.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+---
+
+As per usual, the rest of what you're looking for can be found on [**Fleshgod Apocalypse**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/170), primed for mapping thanks to our ultra-convenient beatmap templates. It's about time your maps reached the infamous black difficulty icon, right?
+
+And that's all for now! Sticking to the trend of artists who've grown in the osu! community through [Project Loved](/wiki/Project_Loved), our next artist is one you might be familiar with if you're an avid fan of unbelievable [WhiteCat](https://osu.ppy.sh/users/4504101) and [Vaxei](https://osu.ppy.sh/users/4787150) DT plays.
```
Comment (Nivalyx) [inline news/2021-06-02-new-featured-artist-fleshgod-apocalypse.md:77]: hmm, i'm not entirely sure about the context here but i think in general the last sentence on this paragraph here would be better off to be written like this tho :

`...if you're an avid fan of unbelievable DT plays from the likes of [WhiteCat](https://osu.ppy.sh/users/4504101) and [Vaxei](https://osu.ppy.sh/users/4787150).`

Comment (pishifat) [reply news/2021-06-02-new-featured-artist-fleshgod-apocalypse.md:77]: verytrue

**Review by peppy — APPROVED** [review]

**Comment by gagahpangeran** [comment]
There's some issue with github itself.

https://www.githubstatus.com/

**Pull request merged**